README.md

```diff
@@ -14,22 +14,50 @@ npm install --save ilp-plugin-lightning
 ## Usage
 
 This plugin can be used with the [`ilp`](https://github.com/interledgerjs/ilp) client module or the [`ilp-connector`](https://github.com/interledgerjs/ilp-connector).
+See the [Ledger Plugin Interface](https://github.com/interledger/rfcs/blob/master/0004-ledger-plugin-interface/0004-ledger-plugin-interface.md) for documentation on available methods.
+
+A minimal way to test your setup:
+* Install LND as explained on https://github.com/lightningnetwork/lnd/blob/master/docs/INSTALL.md
+* Run `lnd --bitcoin.active --bitcoin.testnet --debuglevel=debug --neutrino.active --neutrino.connect=faucet.lightning.community --no-macaroons`
+* Once LND is running, you can try the following script (edit the `lndTlsCertPath` and `peerPublicKey` variables accordingly):
 
 ```js
-const PluginLightning = require('ilp-plugin-lightning')
-const plugin = new PluginLightning({
-  lndUri: 'localhost:11009', // lnd rpc URI
-  // Peer Details
-  rpcUri: 'https://peer.example/rpc',
+const ObjStore = require('ilp-plugin-payment-channel-framework/test/helpers/objStore')
+const PluginLightning = require('.')
+const client = new PluginLightning({
+  server: 'btp+ws://:pass@localhost:9000',
+
+  maxBalance: '1000000', // max allowed balance in Satoshis
```

Comment:
This description could be more helpful. What exactly is the "max allowed balance"?